tests/test_pipeline.py

```diff
@@ -431,12 +439,9 @@ def test_sweep_pipeline_with_save_all_using_mpi(
     assert len(serial_log_files) == 1
     assert len(parallel_log_files) == 1
 
+    log_contents = _get_log_contents(serial_log_files[0])
+    mpi_log_contents = _get_log_contents(parallel_log_files[0])
 
-"""
-#   Something weird going on here with the logs 
-
-    mpi_log_contents = _get_log_contents(log_files[1])
-    log_contents = _get_log_contents(log_files[0])
```

Comment:
Well, my bad, should've added a new commit here or modified the existing one - didn't read the commit message. Usually, people [write good commit messages](https://cbea.ms/git-commit/) with the relevant changes, but we've not been strict about it. I'd simply "squash and merge" from the button down below and write one good commit message.

Reply:
No worries, it's a benign thing really, but since it's such a simple case it's also a good example for learning!

Thanks for the link on writing good commit messages, I tend to use commit messages as places to put useful info in case I were to refer back to them in the future, but I'm sure I could improve mine :muscle: